api/proto/teleport/legacy/types/types.proto

```diff
@@ -3153,6 +3155,90 @@ message WebSessionFilter {
   string User = 1 [(gogoproto.jsontag) = "user"];
 }
 
+// SAMLSessionData contains data for a SAML session.
```

Comment:
Maybe include the link to the crewjam object this is supposed to represent, just for reference?